functions/composer/composer_storage_trigger.py

```diff
@@ -21,7 +21,7 @@
 import google.oauth2.credentials
 import google.oauth2.service_account
 import requests
-
+from google.oauth2 import id_token
```

Comment:
I think this import line needs to be put right after line 20, and there should be 2 empty lines between `import requests` and `IAM_SCOPE=...`

Reply:
Updated